core/variables.js

```diff
@@ -232,8 +232,9 @@ Blockly.Variables.generateUniqueName = function(workspace) {
  * @param {function(?string=)=} opt_callback A callback. It will
  *     be passed an acceptable new variable name, or null if change is to be
  *     aborted (cancel button), or undefined if an existing variable was chosen.
+ * @param {?string} opt_type Optional type of variable, like 'string' or 'list'.
```

Comment:
Hm. Yes, I'll include that '' can be passed in. But I don't want top specify that if nothing is passed in, opt_type will be set to ''. Because that is done in Blockly.Workspace.createVariable(), not here.

Done.